apps/designer/app/routes/rest/tree.$projectId.ts

```diff
@@ -1,7 +1,6 @@
 import { type LoaderFunction } from "@remix-run/node";
 import type { Tree } from "@webstudio-is/react-sdk";
-import * as db from "~/shared/db";
-
+import * as db from "@webstudio-is/project";
```

Comment:
db namespace now makes less sense because its kinda only project part, not all of it

Reply:
Yes, it is just a temporary name. It won't stay like this.

Reply:
Do you have any suggestion on the name? 
`projectdomain`?

![E9s6ooLXsAAKmUP](https://user-images.githubusercontent.com/5845193/189841945-77956ffe-25aa-4043-994a-667177822073.jpg)

Reply:
maybe just destructure the variables we use directly, without using this pattern `* as db` pattern, which is bad for tree-shaking anyways (I guess)

Reply:
alternative is to import {db} from 'package', so that db is a namespace, not an import *